heartbeat/jira

```diff
@@ -0,0 +1,266 @@
+    if jira_monitor; then
+        ocf_log info "Resource is already running"
+        return $OCF_SUCCESS
+    fi
+
+    # Starting Jira
+    /bin/su -m $jira_user -c "$jira_installation/bin/startup.sh &> /dev/null"
+
+    # Verify jira is running
+    while ! jira_monitor; do
+        ocf_log debug "Resource has not started yet, waiting"
+        sleep 1
+    done
+
+    # only return $OCF_SUCCESS if _everything_ succeeded as expected
+    return $OCF_SUCCESS
+}
+
+# Stop
+jira_stop() {
+    local rc
+
+    # exit immediately if configuration is not valid
+    jira_validate_all || exit $?
+
+    jira_monitor
+    rc=$?
+    case "$rc" in
+        "$OCF_SUCCESS")
+            # Currently running. Normal, expected behavior.
+            ocf_log debug "Resource is currently running"
+            ;;
+        "$OCF_NOT_RUNNING")
+            # Currently not running. Nothing to do.
+            ocf_log info "Resource is already stopped"
+            return $OCF_SUCCESS
+            ;;
+    esac
+
+    # sTOPPING Jira
+    /bin/su -m $jira_user -c "$jira_installation/bin/shutdown.sh &> /dev/null"
+
+    # Verify jira is stopped
+    while jira_monitor; do
+        ocf_log debug "Resource has not stopped yet, waiting"
+        sleep 1
+    done
+
+    # only return $OCF_SUCCESS if _everything_ succeeded as expected
+    return $OCF_SUCCESS
+
+}
+
+# Monitor
+jira_monitor() {
+    local rc
+
+    # exit immediately if configuration is not valid
+    jira_validate_all || exit $?
+
+    # Check if Tomcat is running
+    IS_TOMCAT_RUNNING=$(pgrep -f "Dcatalina.home=${jira_installation}")
+
+    if [ -n "$IS_TOMCAT_RUNNING" ] ; then
+      # Is jira working
+      if $(curl -s ${statusurl} | grep '{"state":"RUNNING"}' > /dev/null) ; then
+        rc=0
+      else
+        # Jira has a problem
+        rc=2
+      fi 
+    else
+      # Tomcat is stopped (and Jira)
+      rc=1
+    fi
+
+    case "$rc" in
+        0)
+            rc=$OCF_SUCCESS
+            ocf_log debug "Resource is running"
+            ;;
+        1)
+            rc=$OCF_NOT_RUNNING
+            ocf_log debug "Resource is not running"
+            ;;
+        *)
+            ocf_log err "Resource has failed"
+            exit $OCF_ERR_GENERIC
+    esac
+
+    return $rc
+}
+
+# Validat All
+jira_validate_all() {
+
+    # Check if java is installed
+    if ! [ -d $OCF_RESKEY_java_home ]; then
+      ocf_log err "$OCF_RESKEY_java_home does not exist. \
+      Please ensure that Java is installed and configured correctly"
+      exit $OCF_ERR_INSTALLED
+    fi
+
+    # Check if JIRA installation directory exists 
+    if ! [ -d $OCF_RESKEY_jira_installation ]; then
+      ocf_log err "$OCF_RESKEY_jira_installation does not exist."
+      exit $OCF_ERR_INSTALLED
+    fi
+
+    return $OCF_SUCCESS
+}
+
+# Meta-data
+jira_meta_data(){
+    cat <<EOF
+<?xml version="1.0"?>
+<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
+<resource-agent name="jira" version="0.1">
```

Comment:
There shouldn't be a version attribute here. This is why the CI check fails, so if you remove the version here I think it should pass.

Reply:
Fixed, thanks!